### ISSUE TITLE
Regarding the data division of memotion7k

### ISSUE THREAD

**Issue by PotatoDog1669** [ckoutlis/memefier#2]
hi,
    May I ask how the training set and test set of memotion7k are divided?In the provided dataset link, only the training set is available. May I ask where I can obtain the test set?

**Comment by ckoutlis** [comment]
Hello!
Thank you for your question!
Training and validation sets are taken from [kaggle](https://www.kaggle.com/datasets/williamscott701/memotion-dataset-7k) and are randomly split 90-10 (cf. [these lines of code](https://github.com/ckoutlis/memefier/blob/6b96fd81dea92e4bda7e80c628fa50c7959fa352/src/utils.py#L103C5-L108C36)).
The test set is taken from the corresponding competition page in [CodaLab](https://competitions.codalab.org/competitions/20629), specifically the data and ground truth are available through this [Google Drive link](https://drive.google.com/drive/folders/13Rz7r6JZ4jmsZ_zceJmL3OsHNqJIaiYN?usp=sharing%20). The preparation of the test set is conducted [here](https://github.com/ckoutlis/memefier/blob/6b96fd81dea92e4bda7e80c628fa50c7959fa352/src/utils.py#L110C5-L126C41).
We have updated the README file accordingly.

**Comment by ckoutlis** [comment]
Did you manage to aqcuire the data and run the code @PotatoDog1669 ? Can we close the issue?

**Comment by PotatoDog1669** [comment]
> Did you manage to aqcuire the data and run the code [@PotatoDog1669](https://github.com/PotatoDog1669) ? Can we close the issue?

I'm really sorry, I forgot to reply. I have successfully obtained the data. Thank you very much. You can close it now.
